components/staticFeeds/notifications.json

```diff
@@ -1,4 +0,0 @@
-{"notifications" :
```

Comment:
:question: Is the removal of this file intended?

Reply:
```
docs\messaging-implementation.md
  101:1-101:156  warning  Link to unknown file: `components/staticFeeds/notifications.json`  missing-file  remark-validate-links
```

https://ci.appveyor.com/project/vertein/uportal-app-framework-p1ta9/build/1.0.40#L219